returnn/dataset.py

```diff
@@ -90,3 +93,56 @@ def run(self):
 
             self.out_mean.set(total_mean)
             self.out_std_dev.set(numpy.sqrt(total_var))
+
+
+class SpeakerLabelHDFFromBlissJob(Job):
+    """
+    Extract speakers from a bliss corpus and create an HDF file with a speaker index
+    matching the speaker entry in the corpus speakers for each segment
+    """
+
+    def __init__(self, bliss_corpus, returnn_root=None):
+        """
+        :param bliss_corpus: bliss XML corpus where the speakers and segments are taken from
+        :param returnn_root: used to import SimpleHDFWriter from a specific path if not installed in the worker env
+        """
+        self.bliss_corpus = bliss_corpus
+        self.returnn_root = returnn_root
+        self.out_speaker_hdf = self.output_path("speaker_labels.hdf")
```

Comment:
add: `cached=True`?

Reply:
As we discussed somewhere else the Sisyphus internal caching only works for rasr related things, and those here are not. So I think there is no need. HDF caching will be done by RETURNN anyways.